twitter/favorites.go

```diff
@@ -31,6 +31,12 @@ type FavoriteListParams struct {
 	IncludeEntities *bool  `url:"include_entities,omitempty"`
 }
 
+// FavoriteCreateParams are the parameters for FavoriteService.Create.
+
+type FavoriteCreateParams struct {
```

Comment:
There shouldn't be a blank line between the comment and the type. Otherwise godoc will not pick it up.